### PR TITLE
Fix Dominion Energy

### PR DESCRIPTION
**Pull request by stephengroat**

### CODE REVIEW

File: _data/utilities.yml [hunk 1]
```diff
@@ -73,9 +73,8 @@ websites:
       tfa: No
 
     - name: Dominion
```
Comment (jamcat22) [inline _data/utilities.yml]: It appears they've [rebranded](https://www.dominionenergy.com/NewBrand) as "Dominion Energy", so I think we should update their name to match that.

File: _data/utilities.yml [hunk 2]
```diff
@@ -73,9 +73,8 @@ websites:
...
-      url: https://www.dom.com/
-      twitter: DomVAPower
-      facebook: dominionvirginiapower
+      url: https://www.dominionenergy.com/
+      twitter: dominionenergy
```
Comment (jamcat22) [inline _data/utilities.yml]: It appears that this account is solely used as a corporate news Twitter, and it doesn't look like they reply to tweets. Should we use a different account?

**Review by jamcat22 — CHANGES_REQUESTED** [review]

File: _data/utilities.yml (hunk 2 above)
Comment (stephengroat) [reply _data/utilities.yml]: there isn't any other official account linked on their website

**Review by Carlgo11 — APPROVED** [review]

File: _data/utilities.yml (hunk 2 above)
Comment (Carlgo11) [reply _data/utilities.yml]: chrome_2017-06-22_20-30-11
There are actually a few replies from the account.

**Review by jamcat22 — APPROVED** [review]

**Pull request merged**